app/javascript/packages/document-capture/services/upload.tsx

```diff
@@ -1,22 +1,20 @@
 import { FormError } from '@18f/identity-form-steps';
```

Comment:
I think we should use `.ts` for the file extension here, since there is no JSX/TSX in this file.